devtools/compat/.gitignore

```diff
@@ -0,0 +1,4 @@
+__pycache__
```

Comment:
I just looked at `devtools/`. It was originally used for the internal `cargo x` tooling in Rust.  The `compat` workflow is currently in py and would cause confusion to some.  Would you consider moving compat to `testsuite` dir? I apologize for failing to spot this earlier.

Reply:
will do